source/developer/fx-guidelines.rst

```diff
@@ -459,6 +459,38 @@ All inputs such as textareas should behave consistently, if the default behaviou
         :alt: Edit header modal
         :width: 500 px
 
+Testing Checklist 
+************************
+
+In addition to above guidelines, the below provides a concrete checklist of mistakes to watch for when reviewing proposed product changes. 
+
+User Experience Checklist 
+------------------------------
+
+**1\) Is the WHITESPACE next to icons SUFFICIENTLY SPACED?**
```

Comment:
we should probably get consistent with list labelling: `1\)` used here, `1 -` used in SAML docs.